server/src/v2/middlewares/verify.js

```diff
@@ -0,0 +1,82 @@
+import jwt from 'jsonwebtoken';
+import pool from '../models/configDB';
+
+export default class Middleware {
```

Comment:
@JackieBinya rename this Middleware class to something more descriptive like AuthMiddleware since there are other middlewares in the application